src/goto-programs/goto_check.cpp

```diff
@@ -252,44 +256,19 @@ void goto_checkt::overflow_check(
 }
 
 void goto_checkt::input_overflow_check_int(
-  int width,
-  int limit,
+  const std::string width,
```

Comment:
Why not use the bigint here?

Reply:
Because we do not compare the `width` with the `limit` in this function. Instead, we just need to confirm which of 8/16/32/64 is the `width` of the variable

Reply:
But I do agree we should uniform the format. I have changed it to `BigInt`